### DISCUSSION TITLE
Support for password authentication was removed. Please use a personal access token instead

### DISCUSSION THREAD

**Discussion by guideX** [community/community#42562] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

I tried using the token as the password already, but it still says "Support for password authentication was removed. Please use a personal access token instead", and I'm unable to authenticate to Github anymore, all my code is at a standstill for the last week.

**Comment by airtower-luna** [comment]
In my experience that message appears when whatever you send is *not* a valid token. Obviously I can't say what exactly is going wrong, but double-check there's nothing messing with sending the token (copy&paste error, mistype, stored credentials, expired token, …).

**Comment by guideX** [comment]
I never figured this out, is there anything I can do? I've tried like everything, pat nor ssh nor passwords are working, I cannot authenticate
